sys/socket.txt

```diff
@@ -268,6 +274,12 @@ ifreq_SIOCGIFINDEX {
 	pad		array[const[0, int8], 20]
 } [packed]
 
+
+ifreq_SIOCETHTOOL {
+	ifr_ifrn	devname
+	ifru_data	ptr[inout, ethtool_cmd_u]
```

Comment:
Add padding to make the struct size the same as `ifreq`. Like it's done for `ifreq_SIOCGIFINDEX`.